src/ExtensionsSample/ExtensionsSample.csproj

```diff
@@ -17,7 +17,6 @@
   <ItemGroup>
     <ProjectReference Include="..\Sample.Extension\Sample.Extension.csproj" />
     <ProjectReference Include="..\WebJobs.Extensions.CosmosDB\WebJobs.Extensions.CosmosDB.csproj" />
-    <ProjectReference Include="..\WebJobs.Extensions.Http\WebJobs.Extensions.Http.csproj" />
```

Comment:
The sample has some http functions - how are they working if you've removed this?

Reply:
Just checked. No HTTP samples there, which does make sense as we don't really have a listener supporting HTTP.